### ISSUE TITLE
hint: for better images in frontend>event and frontend>venue

### ISSUE THREAD

**Issue by jojo12** [Heklaterriol/JEM-Project#179]
why do not change in core /or as override:

We have users, who use better and bigger images in event details, for this is the flyer2 in output.class.php which preferes the original images than the small images.
for this: exchange in site/views/tmpl/responsive/default.php "JemOutput::flyer" with "JemOutput::flyer2"

**Comment by mckillo** [comment]
I can see those 2 functions in components\com_jem\classes\output.class.php file:

Line 1096 - static public function flyer($data, $image, $type, $id = null) ...... with 21 call from JEM code.
Line 1162 - static public function flyer2($data, $image, $type, $id = null) .... with 0 call from JEM code.

I think that the best solution is to have a option in JEM setting to custom size (resolution & ratio)....but I will have to see the example to compare it and see possible solutionwith a param.
Any example?

**Comment by jojo12** [comment]
flyer 2 was created from stevent who was using old JEM extensif in ugenda.nl. I fear, he will not come back :-(
he had in eventdetails very big clear images (depends on the JEM settings when uploading. For the tables the images were normal small. With flyer2 in output.class.php it was possible to  make overrides for such using.
look in demo.joomlaeventmanager.net (look the link of the used images jem/events/small/.. or jem/events/.. 
same with venues.

**Comment by mckillo** [comment]
1 think that If it's a particular case (for one user), I think that its better to do override of view and delete the code of flyer2.
If we create a new option to select between small or big (plus width define) thumb image in global setting:

![JEM - Draft desgin of image setting for Thumbs](https://user-images.githubusercontent.com/2765316/229103860-955ec1e9-bb84-4f56-859b-ed65cbc1aca2.jpg)

it will be for all view.
In this case, there is created a files for other big-thumb

**Comment by jojo12** [comment]
Be conscient that flyer2 / images originals are essential for a lot in modules too!!!

**Comment by mckillo** [comment]
I agree you,modules are affected too and output.class.php not overrides.
I think that flyer2 isn't priority, I can be change new version. So, let's see with more time to see possible solutions.

I have spend a lot of time to install-update script ... a lot hours in front 'bicho' (computer). For me, this issue have a first proposal with new params for image... and we give it a spin/around
